bundle/Resources/config/services.yml

```diff
@@ -18,3 +25,16 @@ services:
         parent: ezplatform.installer.db_based_installer
         tags:
             - {name: ezplatform.installer, type: netgen-media-clean}
+
+    netgen_site_installer.media_site_remote_clean:
+        class: Netgen\Bundle\SiteInstallerBundle\Installer\NetgenRemoteMediaSiteInstaller
+        public: false
+        calls:
+            - [ setInstallerDataPath, [ "%kernel.project_dir%/vendor/netgen/media-site-data/netgen-media-clean" ] ]
+            - [ setStoragePath, [ web/var/site/storage ] ]
+            - [ setRepository, [ "@ezpublish.api.repository" ] ]
+            - [ setFieldTypesToMigrate, [ "%netgen_site_installer.remote_media.field_types_to_migrate%" ] ]
+            - [ setExcludedFieldIdentifiers, [ "%netgen_site_installer.remote_media.excluded_field_identifiers%" ] ]
+        parent: ezplatform.installer.db_based_installer
```

Comment:
Shouldn't parent be `netgen_site_installer.media_site_clean` so it matches the class? In that case, you wouldn't need to specify `setInstallerDataPath` and `setStoragePath` too.

Reply:
Changed in 75b6f1d8173c4d8efd5101c63e99b228c63ecabb
Also fixed a bug, as `installerDataPath` is private in the parent class.

Reply:
We can make it protected, since it's obviously useful. Just add a comment on why it's protected and not private.

Reply:
Changed in 42650eb